1.1 build error

Building 1.1 release fails with this error:

build_store.c: In function ‘data_src_cell_toggled’:
build_store.c:385:12: error: two or more data types in declaration specifiers
  385 |   gboolean bool;
      |            ^~~~
build_store.c:385:3: warning: useless type name in empty declaration
  385 |   gboolean bool;
      |   ^~~~~~~~
In file included from /usr/include/unicode/umachine.h:52,
                 from /usr/include/unicode/utypes.h:38,
                 from /usr/include/unicode/ucnv_err.h:88,
                 from /usr/include/unicode/ucnv.h:52,
                 from /usr/include/libxml2/libxml/encoding.h:31,
                 from /usr/include/libxml2/libxml/parser.h:810,
                 from build_store.c:35:
build_store.c:388:60: error: expected expression before ‘_Bool’
  388 |   gtk_tree_model_get(GTK_TREE_MODEL(gui->list), &iter, 0, &bool, 1, &type, -1);
      |                                                            ^~~~
build_store.c:390:12: error: expected expression before ‘_Bool’
  390 |        0, !bool && (type != DATA_SRC_CDDB || gui->model->cddb_mask), -1);
      |            ^~~~
build_store.c:386:7: warning: unused variable ‘type’ [-Wunused-variable]
  386 |   int type;
      |       ^~~~
make[3]: *** [Makefile:717: aqualung-build_store.o] Error 1

--- a/src/build_store.c
+++ b/src/build_store.c
@@ -382,12 +382,12 @@ data_src_cell_toggled(GtkCellRendererToggle * cell, gchar * path, gpointer data)
 	data_src_gui_t * gui = (data_src_gui_t *)data;
 
 	if (gtk_tree_model_get_iter_from_string(GTK_TREE_MODEL(gui->list), &iter, path)) {
-		gboolean bool;
+		gboolean value;
 		int type;
 
-		gtk_tree_model_get(GTK_TREE_MODEL(gui->list), &iter, 0, &bool, 1, &type, -1);
+		gtk_tree_model_get(GTK_TREE_MODEL(gui->list), &iter, 0, &value, 1, &type, -1);
 		gtk_list_store_set(GTK_LIST_STORE(gui->list), &iter,
-				   0, !bool && (type != DATA_SRC_CDDB || gui->model->cddb_mask), -1);
+				   0, !value && (type != DATA_SRC_CDDB || gui->model->cddb_mask), -1);
 	}
 }
 

